tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -75,6 +75,30 @@ def testee(a: cases.IJKField) -> cases.IJKField:
     cases.verify_with_default_data(cartesian_case, testee, ref=lambda a: a)
 
 
+def test_interface_kwargs(fieldview_backend):
```

Comment:
Should use the `cartesian_case` fixture and `allocate`/`verify` instead of the old `fieldview_backend`